superset-embedded-sdk/src/index.ts

```diff
@@ -86,6 +88,7 @@ export async function embedDashboard({
   dashboardUiConfig,
   debug = false,
   iframeTitle = "Embedded Dashboard",
+  iframeSandboxExtras = []
```

Comment:
The docs help immensely. Curious others thoughts here, but I wouldn't mind seeing the suggested options here as comments (not enabled, just commented-out lines so it's trivial for people to just turn them on).